### PR TITLE
Support vaccine studies in EMIS

### PR DESCRIPTION
**Pull request by inglesp**

### CODE REVIEW

File: cohortextractor/emis_backend.py [hunk 1]
```diff
@@ -1165,6 +1165,10 @@ def quote(value):
         return str(value)
 
     value = str(value)
+
+    if re.match(r"^\w+ \+ interval '\d+' (day|month|year)$", value):
+        return value
+
```
Comment (evansd) [inline cohortextractor/emis_backend.py:1173]: I don't know enough of the context here to comment really. I'm a bit surprised this all works but I'll trust you that it does.

Do you need to support negative intervals?

Is the idea that this is temporary and we'll move to the existing `<date> + 10 days` syntax at some point?

**Review by evansd — APPROVED** [review]
Happy to approve this but I don't fully understand what the date parsing changes are doing

File: cohortextractor/emis_backend.py (hunk 1 above)
Comment (inglesp) [reply cohortextractor/emis_backend.py:1173]: Was sure I'd written an explanation / note of apology somewhere, sorry.

Yes, the idea is this is temporary.  Supporting `<date> + 10 days` will presumably come for free (with the code to actually generate the SQL as a rounding error) once we've done the big refactor.

Comment (evansd) [reply cohortextractor/emis_backend.py:1173]: :+1: Makes sense

**Pull request merged**